packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -689,7 +698,9 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 		const gcData: IGarbageCollectionData = { gcNodes: {} };
 		for (const [localId, storageId] of this.redirectTable) {
 			assert(!!storageId, 0x390 /* Must be attached to get GC data */);
-			gcData.gcNodes[this.getBlobGCNodePath(localId)] = [this.getBlobGCNodePath(storageId)];
+			if (localId !== storageId) {
```

Comment:
Add a comment what this check means and why the outbound routes are empty.

Reply:
Is it to clear out the old mapping?

Reply:
It's because there are no references from blob ids. blob can have inbound routes (references) from DDSes but no outbound routes.
Before this change, local id -> storage id was considered a reference so the outbound routes was not empty, not anymore.

Reply:
So you have to have an entry like this to represent the node?  It's not enough to just have some other GC data list this local ID in their outbound route?  Is this how you do leaf nodes elsewhere as well?